app/controllers/candidate_interface/unsubmitted_application_form_controller.rb

```diff
@@ -9,6 +9,7 @@ def before_you_start; end
     def show
       @application_form_presenter = CandidateInterface::ApplicationFormPresenter.new(current_application)
       @application_form = current_application
+      @application_cache_key = "#{@application_form.cache_key_with_version}-#{Feature.maximum(:updated_at)}"
```

Comment:
So if the application forms updated at is greater than the `#{Feature.maximum(:updated_at)}` then it to ignore the cache?

Reply:
> So if the application form or any feature flags updated_at is greater than the cache's created_at then it ignores the cache?

Exactly — it'll generate this key at every page load, and if neither timestamp has changed it'll just use the fragment in the cache (everything enclosed by the `cache` block in the template) when rendering the view. Otherwise it'll process the whole template.